lib/libutils/ext/include/confine_array_index.h

```diff
@@ -0,0 +1,102 @@
+// Copyright 2019 The Fuchsia Authors. All rights reserved.
```

Comment:
I think `/* SPDX-License-Identifier: BSD-3-Clause */` applies here.

Reply:
Well spotted, I've checked it word for word and it matches. Only difference is `*` instead of numbers in the clauses. I'll update.